target/linux/ramips/mt7620/base-files/etc/board.d/02_network

```diff
@@ -207,6 +207,12 @@ ramips_setup_interfaces()
 		ucidef_add_switch "switch0" \
 			"1:lan:3" "2:lan:4" "3:lan:1" "4:lan:2" "0:wan" "6@eth0"
 		;;
+	tplink,archer-c5-v4)
+		ucidef_add_switch "switch0"
+		ucidef_add_switch_attr "switch0" "enable" "0"
+		ucidef_add_switch "switch1" \
+			"0:lan:4" "1:lan:3" "2:lan:2" "3:lan:1" "4:wan" "5@eth0"
```

Comment:
Should be `7@eth0`

Reply:
Not for the new driver. CPU is at port 5. Port 7 is missing.

Reply:
Does the luci switch page show CPU port link status correctly?

Reply:
![Screenshot_20210322_020202](https://user-images.githubusercontent.com/836788/111942807-e07c8200-8ab2-11eb-92b1-516a1181dc75.png)

I looks OK, although CPU port has no link. I get similar info from swconfig:

```
# swconfig dev switch1 port 5 show
...

I don't know what the old driver returned. Anyway all labels match the corresponding case label.
ifInBroadcastPkts                   : 26
ifOutDiscards                       : 0

        pvid: 1
        link: ???
```